resources_pages/install_ds_stack_mac.md

```diff
@@ -260,13 +254,15 @@ you can replace `conda install` with `conda install -y`.
 
 ## JupyterLab setup
 
-We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
+We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. We will also use Jupyter Book to create publication ready documents. Install them via the following commands:
 
 ```bash
-conda install nodejs=15.*
-conda install -c conda-forge jupyterlab jupyterlab-git jupytext
+conda install jupyterlab-git jupytext jupyter-book jupyterlab-spellchecker
```

Comment:
I just tried this on a fresh Mac. I got the following warning:
```
The following packages will be SUPERSEDED by a higher-priority channel:

  nbconvert          conda-forge/osx-64::nbconvert-6.1.0-p~ --> conda-forge/noarch::nbconvert-5.6.0-py_0
```
So while we already get nbconvert 6.1, running this line downgrades it to 5.6. After that, running the next command (the one using pip) did not work, complaining that:
```
WARNING: nbconvert 5.6.0 does not provide the extra 'webpdf'
```
and finally failed, giving the following error:
```
-bash: pyppeteer-install: command not found
```

Reply:
@joelostblom I tried the solution you provided earlier by running the following
```
pip uninstall nbconvert
pip install -U "nbconvert[webpdf]"
```
And again, it solved the problem. Only after running the above the `pyppeteer-install` worked successfully.

Reply:
Thanks, I also ran into this and I am trying to find the best solution (potentially installing pyppeteer separately and jupytext from GH main)